ArduCopter/mode.cpp

```diff
@@ -461,23 +461,18 @@ void Copter::notify_flight_mode() {
 }
 
 // get_pilot_desired_angle - transform pilot's roll or pitch input into a desired lean angle
-// returns desired angle in centi-degrees
-void Mode::get_pilot_desired_lean_angles_cd(float &roll_out_cd, float &pitch_out_cd, float angle_max_cd, float angle_limit_cd) const
+// returns desired angle in radians
+void Mode::get_pilot_desired_lean_angles_rad(float &roll_out_rad, float &pitch_out_rad, float angle_max_rad, float angle_limit_rad) const
 {
-    if (!rc().has_valid_input()) {
-        roll_out_cd = 0.0;
-        pitch_out_cd = 0.0;
+    // throttle failsafe check
+    if (copter.failsafe.radio || !rc().has_ever_seen_rc_input()) {
```

Comment:
ah, this looks like a regression from a change that PeterB put in a few weeks ago